srcpkgs/passff-host/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'passff-host'
+pkgname=passff-host
+version=1.2.1
+revision=1
+hostmakedepends="make"
+makedepends=""
+depends="python3"
+short_desc="Host app for the WebExtension PassFF"
+maintainer="Colin Reeder <colin@vpzom.click>"
+license="GPL-2.0"
+homepage="https://github.com/passff/passff-host"
+distfiles="https://github.com/passff/passff-host/releases/download/$version/passff.json https://github.com/passff/passff-host/releases/download/$version/passff.py"
+checksum="cd871bcf29d71e53f986fb74b7f2b1867151ba2b2a27318044c7bfaa38827282 6db5c9c435bfe8efc0f626e38144eff4c045d90d8508a295c990abbae411ac6c"
+skip_extraction="passff.json passff.py"
+create_wrksrc=yes
```

Comment:
Move this line up to before `depends=`. You should install `xtools` and run `xlint srcpkgs/passff-host/template` to see what and how to fix in your template.